### PR TITLE
front - adds new commands to format tags in cypress

### PR DESCRIPTION
**Pull request by basilelegal**
#1826 adds 2 new function to format tags and a new cypress command to use instead of the usual `cy.get`

- [x] Code is clear enough

### CODE REVIEW

**Comment by basilelegal** [comment]
errors in tests will be fixes by #1832

File: functionnal_tests/cypress/support/generic_selector_commands.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+const SELECTORS = {
+  CONTENT_FRAME: 'contentFrame',
+  CONTENT_IN_LIST: 'contentInList',
+  WORKSPACE_MENU: 'workspaceMenu'
+}
+
+
+const TAGS = {
```
Comment (Skylsmoi) [inline functionnal_tests/cypress/support/generic_selector_commands.js]: you can use this syntax:
```
const TAGS = {
  [SELECTORS.CONTENT_FRAME]: '...'
}
```

File: functionnal_tests/cypress/support/generic_selector_commands.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    Object.entries(attrs).forEach(([key, value]) => tag += `[${key}="${value}"]`)
+  }
+  return tag
+}
+
+Cypress.Commands.add('getTag', ({selectorName, params = {}, attrs = null}) => {
+  return cy.get(formatTag({selectorName: selectorName, params: params, attrs: attrs}))
+})
+
+export { SELECTORS, formatTag, reverseTag }
```
Comment (Skylsmoi) [inline functionnal_tests/cypress/support/generic_selector_commands.js]: missing new line

**Comment by Skylsmoi** [comment]
I'm confused @basilelegal.
You say errors in tests will be fixed by #1832 but #1832 says it needs #1831 (this PR) to be merged.
It's feels like circular import

**Comment by basilelegal** [comment]
The cypress tests failling (random fails) are prior to this pr, this pr brings only new function and don't change any test. Tests are not to be fixed here.

#1832 fix the tests but is based on #1831, so we need to merge 1831 first.

**Comment by Skylsmoi** [comment]
ok I understand

**Pull request merged**